samples/nrf9160/serial_lte_modem/Kconfig

```diff
@@ -32,15 +32,14 @@ choice
 		-  UART 2
 	config SLM_CONNECT_UART_0
 		bool "UART 0"
+	  select UART_0_NRF_HW_ASYNC
```

Comment:
These should be aligned with `bool "UART 0"`. Why does UART 0 require a different configuration than UART 2?

Reply:
We cannot set UART_0_NRF_UARTE, which will result in CMake warning. According to @nordic-krch this is due to nRF9160 does not have UART0, only UARTE0, so manual configuration is not allowed, but automatic configuration is always done.

Also while using UART_0 flow control is not enabled.